### DISCUSSION TITLE
WSA Error Accidentally Uninstalled

### DISCUSSION THREAD

**Discussion by technicks07** [MustardChef/WSABuilds#183] [General]
Hi  I Have a question i installed WSA and after install I accidentally uninstalled the WSA and when I install it again and run the Run.bat i have a error now.. and i dont know how to fix this.. help me pls. 
![Capture](https://github.com/MustardChef/WSABuilds/assets/46170712/57c4d747-cb9c-4783-a865-e62233240889)

Add-AppxPackage : Deployment failed with HRESULT: 0x80073CF6, Package could not be registered.
error 0x80073D26: While processing the request, the system failed to register the windows.service extension due to the
following error: The package installation failed because a different version of the service is installed. Try
installing a newer version of the package.
.
NOTE: For additional information, look for [ActivityId] d9cb1310-e619-0002-7225-cbd919e6d901 in the Event Log or use
the command line Get-AppPackageLog -ActivityID d9cb1310-e619-0002-7225-cbd919e6d901
At line:1 char:1
+ Add-AppxPackage -ForceApplicationShutdown -ForceUpdateFromAnyVersion  ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (E:\Drive C\Docu...ppxManifest.xml:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand

**Comment by technicks07** [comment]
HElp me @MustardChef  plsss :(

**Comment by MustardChef** [comment]
@technicks07 

Point 1: Please download the latest WSA build as 2303.40000.5.0 is 5+ months old

Point 2: Please run the following before running ``Run.bat``

```powershell
Get-AppxPackage -Name "MicrosoftCorporationII.WindowsSubsystemForAndroid" -AllUsers | Remove-AppxPackage -AllUsers
```

**Reply by technicks07** [reply]
@MustardChef  where can I run thisGet-AppxPackage -Name "MicrosoftCorporationII.WindowsSubsystemForAndroid" -AllUsers | Remove-AppxPackage -AllUsers in the directory of WSA?

**Reply by technicks07** [reply]
![image](https://github.com/MustardChef/WSABuilds/assets/46170712/baae8b14-5b49-493d-a18e-633d9772f7c9)
When i run that code.. nothings happen.

**Reply by technicks07** [reply]
I already download the latest WSA Build for windows 10

**Comment by technicks07** [comment]
![image](https://github.com/MustardChef/WSABuilds/assets/46170712/469e2476-4815-44b9-a548-f0fb23214201)
i still have an error i follow your instruction and this is the error...  @MustardChef

**Reply by MustardChef** [reply]
@technicks07 
Please provide the logs by running:
Get-AppPackageLog -ActivityID 4c3ff12d-e622-0002-b8f4-3f4c22e6d901

**Reply by technicks07** [reply]
@MustardChef i installed the new WSA Build but i have a new problem after showing the loading of Starting Windows Subsystem For Android™ nothing happens I thought the google play store is pop out after that loading but for me nothing is happen. Please Help Me this... thank you so much!
![image](https://github.com/MustardChef/WSABuilds/assets/46170712/2067057f-ba23-4824-8663-4bc41c8b6335)

**Reply by MustardChef** [reply]
@technicks07 
Please provide me with more information:
- Windows Version and Build Number?
- CPU?
- GPU?
- RAM?

**Reply by technicks07** [reply]
Windows 10 Ghost Spectre Super Lite 20H2 Build 19042.985
CPU: Intel i5 7400 
GPU: Nvidia GeForce  Gtx 1050ti
RAM: 16Gb

@MustardChef

**Reply by MustardChef** [reply]
@technicks07 The issue here is the fact that you are using a modified version of Windows 10 which is on 20H2. Your PC Specs are fine, so it may be modified Windows build that is causing this, as they tend to disable features and services that may be required for WSA.
